pkg/event/list.go

```diff
@@ -3,7 +3,8 @@ package event
 import "time"
 
 const (
-	DateFormat                = "2006-01-02"
+	DateFormat                = "2006-01-02T15:04:05.999999Z07:00"
```

Comment:
We should rename this to something more descriptive like `DateFormatRFC3339Micro`. Also, if it's only being used in the postgres repository, it might make sense to move it there.